src/coordinator/generated/proto/rpc/query.proto

```diff
@@ -25,7 +25,12 @@ message WriteOptions {
 
 message Datapoint {
 	int64 timestamp = 1;
```

Comment:
Might be something to be said for adding a new path for sending through fixed resolution datapoints to avoid sending timestamps. OTOH, this will probably change soon to send the raw buffers when possible

Reply:
yes, I wasn't sure what would be the best way to handle this. Maybe rpc.Series can have both datapoints ([]datapoint) and values ([]float64). The remote just populates one of them ?